opre_ops/ops_site/migrations/0002_auto_20210901_2327.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.6 on 2021-09-01 23:27
+
+import os
+
+from opre_ops.settings.local import BASE_DIR
```

Comment:
_[caveat: I know this is a draft PR, you may have been planning to change this anyway!]_

The migrations will run in both local and cloud environments, so I don't think we'll want to pull from local settings. Looks like we could pull this from `settings.common`?

However, I don't see this imported variable used anywhere, so maybe we don't need it?

Reply:
Oh yes! I was using this setting in a test of something and forgot to remove it. If we needed it you'd be correct though :)

Reply:
👍 👍  Thanks!